pkg/controller/channel/controller.go

```diff
@@ -539,19 +539,18 @@ func newVirtualService(channel *channelsv1alpha1.Channel) *istiov1alpha3.Virtual
 				controller.ServiceHostName(controller.ChannelServiceName(channel.Name), channel.Namespace),
 				controller.ChannelHostName(channel.Name, channel.Namespace),
 			},
-			Http: []istiov1alpha3.HTTPRoute{
-				{
-					Rewrite: &istiov1alpha3.HTTPRewrite{
-						Authority: controller.ChannelHostName(channel.Name, channel.Namespace),
-					},
-					Route: []istiov1alpha3.DestinationWeight{
-						{
-							Destination: istiov1alpha3.Destination{
-								Host: destinationHost,
-							},
-						},
-					},
+			Http: []istiov1alpha3.HTTPRoute{{
+				Rewrite: &istiov1alpha3.HTTPRewrite{
+					Authority: controller.ChannelHostName(channel.Name, channel.Namespace),
 				},
+				Route: []istiov1alpha3.DestinationWeight{{
+					Destination: istiov1alpha3.Destination{
+						Host: destinationHost,
+						Port: istiov1alpha3.PortSelector{
+							Number: 80,
```

Comment:
do we already have a constant for this port number (or port name)?

Reply:
updated